ui/src/components/admin/helpers/modelForm.js

```diff
@@ -1,4 +1,10 @@
+// import {getDictionary} from '../dictionary';
 export const isFormReadyToSave = (dirty, errors) => dirty && !('name' in errors);
 
 export const isFormReadyToPublish = (values, dirty, errors) =>
   (values.status !== 'published' || dirty) && Object.keys(errors).length === 0;
+
+// export const fetchModelValidator = async (excludeModels) => {
```

Comment:
Is this stuff meant to be used in a later ticket?

Reply:
oh, no, i can remove this. I changed how this worked. thanks for calling it out.

Reply:
Nope, I changed how i was approaching this issue part way. this code can be removed. Thanks for catching this.